### DISCUSSION TITLE
Font Awesome merging / question mark

### DISCUSSION THREAD

**Discussion by SadE54** [ocornut/imgui#4598] [New Users Build/Link/Run issues ONLY!]
Hi, 

I'm trying to use font awesome merged with the default font.
But  each time I'm trying to display an icon, i 'm getting a question mark instead 🤨
I'm using this code :
![devenv_RLGc7ySlfw](https://user-images.githubusercontent.com/2878768/135299619-77dea0b7-5a1b-4ddb-af48-585c3eb00c43.png)

The font file seems to be loaded correctly (form the file ) and after the build i get this data:
![jVV2jDjsKm](https://user-images.githubusercontent.com/2878768/135299927-755c3fdc-6f03-45b5-96fd-ffa46863762c.png)

I'm not sure but it seems ok (except some zeroed properties ? ) 

Someone has some hint to stop my headache with that ? 😬 (using Imgui 1.84 with dx11)

**Comment by ocornut** [comment]
Start by browsing the Fonts contents in Metrics/Debugger.

**Reply by SadE54** [reply]
Seriously , I started to work with Imgui 2 months ago , and I didn not know there was such a metric debugger integrated  😆
I'm starting to understand the the the icon font was merged to a font that is not the default one,..
btw, i just saw the table is almost empty : 
![MEFlow_Vfb18QeB8A](https://user-images.githubusercontent.com/2878768/135335078-1158160c-cd07-4ac2-bbb5-4f602af9af27.png)
There's only a subset in the free font ...  So I tried to reach an empty glyph . This , over the first reason, no way to display something 😒

**Reply by ocornut** [reply]
> and I didn not know there was such a metric debugger integrated 😆

FYI this is the first thing mentioned (with a screenshot) in the FONTS documentation
https://github.com/ocornut/imgui/blob/master/docs/FONTS.md
I'm not sure how we can improve discoverability more than that...

It's available from both Style Editor and Metrics/Debugger panels.

**Reply by SadE54** [reply]
You're absolutely right , you cannot improve visibility . It's my fault . I guess It's because i firstly discovered a pure text version of the font help file , and was sticked with it !😁

**Comment by PathogenDavid** [comment]
Make sure you aren't hitting the same situation as https://github.com/ocornut/imgui/discussions/4561#discussioncomment-1357349

**Reply by SadE54** [reply]
Thx for the link and trick !
